src/dune_file.mli

```diff
@@ -372,6 +372,16 @@ module Coq : sig
   type Stanza.t += T of t
 end
 
+module CoqPp : sig
+
+  type t =
+    { modules    : string list
```

Comment:
It shouldn't be much effort to make this work using Ordered_set_language. Let me know if you need some help with that.

Reply:
I used `string list` as this is what `ocamlex/menhir` etc... use; I'm not sure there is much benefit from using the OSL, you have between 1-2 mlg declarations in the typical Coq plugin.

Reply:
I used `string list` as this is what `ocamlex/menhir` etc... use; I'm not sure there is much benefit from using the OSL, you have between 1-2 mlg declarations in the typical Coq plugin.